test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('chai');
+const axios = require('axios');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+chai.use(chaiSubset);
+
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+describe('Github Delete Testing', () => {
+  describe('Delete M', async () => {
+    it('Create', async () => {
+      const gist = await object.post('https://api.github.com/gists', { description: 'Promise #1', public: true, files: { 'README.md': { content: 'Promise #1' } } });
```

Comment:
can you create a global const variable to represent the gist to create, so you can use that object in this line and also, in the containSubset

Reply:
done